base/deprecated.jl

```diff
@@ -1722,20 +1722,20 @@ import .Iterators.enumerate
 
 # PR #23640
 # when this deprecation is deleted, remove all calls to it, and replace all keywords of:
-# `payload::Union{CredentialPayload,Nullable{<:AbstractCredentials}}` with
+# `payload::Union{CredentialPayload,Union{Some{<:AbstractCredentials}, Void}}` with
 # `payload::CredentialPayload` from base/libgit2/libgit2.jl
 @eval LibGit2 function deprecate_nullable_creds(f, sig, payload)
-    if isa(payload, Nullable{<:AbstractCredentials})
+    if isa(payload, Union{Some{<:AbstractCredentials}, Void})
```

Comment:
This one is a bit special. This function is called by internal code, so it needs to work with the new framework. Else we would break even more things than needed.